gammapy/estimators/points/core.py

```diff
@@ -612,3 +614,41 @@ def plot_ts_profiles(
             ax.figure.colorbar(caxes, ax=ax, label=label)
 
         return ax
+
+    def recompute_ul(self, n_sigma_ul=2, **kwargs):
+        """Recompute upper limits corresponding to the given value.
+        The pre-computed stat profiles must exist for the re-computation.
+
+        Parameters
+        ----------
+        n_sigma_ul : int
+            Number of sigma to use for upper limit computation. Default is 2.
+        **kwargs : dict
+            Keyword arguments passed to `~scipy.optimize.brentq`.
+
```

Comment:
Please add an example here using `"$GAMMAPY_DATA/tests/spectrum/flux_points/binlike.fits"`

Reply:
Hi @adonath ! How was this file produced? The stat scan values on this seem to be the negative of our current convention

Reply:
Yes, but we have the conversion code here: https://github.com/gammapy/gammapy/blob/master/gammapy/estimators/points/core.py#L234

Doesn't this work?

Reply:
Okay added

Reply:
Implicitly it did not. I added an `np.abs` so that a convex function is passed to `stat_profile_ul_scipy`

Reply:
Thanks! I wonder why it works for the plotting...I can check...